www-servers/lighttpd/lighttpd-1.4.39.recipe

```diff
@@ -124,8 +125,9 @@ BUILD_PREREQUIRES="
 BUILD()
 {
 	aclocal
+	autoconf
 	automake
-	runConfigure ./configure --with-openssl
+	LIBS=-lnetwork runConfigure ./configure --with-openssl
```

Comment:
This can be done from inside configure.ac
https://autotools.io/autoconf/finding.html

something like this:

    AC_SEARCH_LIBS([gethostbyname], [network], [], [AC_MSG_ERROR([unable to find the gethostbyname() function])])